interfaces/cython/cantera/reaction.pyx

```diff
@@ -120,72 +120,102 @@ cdef class ReactionRate:
             return anymap_to_dict(self.rate.parameters())
 
 
-cdef class ArrheniusRate(ReactionRate):
-    r"""
-    A reaction rate coefficient which depends on temperature only and follows
-    the modified Arrhenius form:
-
-    .. math::
-
-        k_f = A T^b \exp{-\tfrac{E}{RT}}
-
-    where ``A`` is the `pre_exponential_factor`, ``b`` is the `temperature_exponent`,
-    and ``Ea`` is the `activation_energy`.
+cdef class _ArrheniusTypeRate(ReactionRate):
```

Comment:
This class needs to be added to `kinetics.rst`

Reply:
Ok. While this is a hidden class, the documentation needs to be added, correct? Is there a way to specify inherited members (by level) in Sphinx?

Reply:
I ended up un-hiding the class in Python, and added it to `kinetics.rst` as requested.